### PR TITLE
Fix/alert policy and time

### PR DESCRIPTION
**Pull request by HoOngEe**
Fixed the problem printing wrong time in the title of alert emails.
Changed the alert policy

### CODE REVIEW

File: monitor/index.ts [hunk 1]
```diff
@@ -29,21 +37,100 @@ const checkDeath = (() => {
...
+  if (multipleNodesSleeping) {
+    state.prevHasProblem = true;
+    sendNotice(
+      new chainErrors.NodeIsSleeping(bestBlockNumber, sleepingNodeIndices),
+      targetEmail
+    );
+  }
+}
+
+function alertWhenAllNodesWakeUp(
```
Comment (sgkim126) [inline monitor/index.ts]: IMO it's not an alert.

File: monitor/index.ts [hunk 2]
```diff
@@ -29,21 +37,100 @@ const checkDeath = (() => {
...
+) {
+  const allNodesNowAwake =
+    sleepingNodeIndices.length === 0 && state.prevHasProblem;
+  if (allNodesNowAwake) {
+    state.prevHasProblem = false;
+    sendNotice(new chainErrors.AllNodesAwake(bestBlockNumber), targetEmail);
+  }
+}
+
+function alertWhenNodesSleepingLong(
```
Comment (sgkim126) [inline monitor/index.ts]: ditto

File: monitor/index.ts [hunk 3]
```diff
@@ -29,21 +37,100 @@ const checkDeath = (() => {
...
+  bestBlockNumber: number,
+  targetEmail: string,
+  state: CheckSealFieldState,
+  sleepingNodeIndices: number[]
+) {
+  for (let idx = 0; idx < state.sleepStreak.length; idx++) {
```
Comment (sgkim126) [inline monitor/index.ts]:
```ts
const idx = 0
```

File: monitor/index.ts [hunk 4]
```diff
@@ -29,21 +37,100 @@ const checkDeath = (() => {
...
+    if (sleepingNodeIndices.includes(idx)) {
+      state.sleepStreak[idx] += 1;
+    } else {
+      state.sleepStreak[idx] = 0;
+    }
+  }
+  const longTermSleepingIndices = [];
+  for (let idx = 0; idx < state.sleepStreak.length; idx++) {
```
Comment (sgkim126) [inline monitor/index.ts]: ditto

Comment (sgkim126) [inline monitor/index.ts]: How about merging this loop with the above loop?

File: monitor/index.ts (hunk 3 above)
Comment (sgkim126) [reply monitor/index.ts]: My mistake. It cannot be const.

File: monitor/index.ts (hunk 1 above)
Comment (HoOngEe) [reply monitor/index.ts]: I changed "alert" to "notify"

File: monitor/index.ts (hunk 2 above)
Comment (HoOngEe) [reply monitor/index.ts]: I changed "alert" to "notify"

File: monitor/index.ts (hunk 4 above)
Comment (HoOngEe) [reply monitor/index.ts]: loop merged

**Pull request merged**